src/adera/machines/links.cpp

```diff
@@ -1,12 +1,63 @@
 #include "links.h"
 
+using namespace osp;
+
 using osp::link::MachTypeReg_t;
 using osp::link::MachTypeId;
 
 namespace adera
 {
 
-MachTypeId const gc_mtUserCtrl = osp::link::MachTypeReg_t::create();
-MachTypeId const gc_mtMagicRocket = osp::link::MachTypeReg_t::create();
+MachTypeId const gc_mtUserCtrl      = MachTypeReg_t::create();
+MachTypeId const gc_mtMagicRocket   = MachTypeReg_t::create();
+MachTypeId const gc_mtRcsDriver     = MachTypeReg_t::create();
```

Comment:
Should these go into a headerfile as "inline" ?

Reply:
It can. I vaguely recall there was something similar to this a long time ago that was originally inline, then you suggested changing it to extern.

Either way, I'd assume there has to be some minor advantages to being declared in just one translation unit.